src/theme/Navbar/MobileSidebar/Layout/styles.module.css

```diff
@@ -1,11 +1,20 @@
 [class*="navbar-sidebar__items"] {
-  height: calc(100% - var(--ifm-navbar-height) - 64px);
+  --swm-navbar-sidebar-items-height: 127px; /* Overall height of additional items in mobile sidebar */
+  height: calc(
+    100% - var(--ifm-navbar-height) - var(--swm-navbar-sidebar-items-height)
+  );
 }
 
 [class*="navbar-sidebar__brand"] {
   padding-left: 20px;
 }
 
+@media (max-width: 997px) {
```

Comment:
In some places you've changed 997 to 996. I think we should unify this. 

Sadly it's not possible to use CSS variables for this 😭

Reply:
Sure! I thought about the same thing, but I forgot to do this 😁 Thanks for mentioning about it!